app.py

```diff
@@ -12,45 +15,210 @@
     initial_sidebar_state="expanded",
 )
 
-st.title("A sneak peek at the stats in Open Targets")
+st.title("A sneak peek at the stats in the Open Targets Platform.")
+st.markdown('##')
 st.markdown("This application is a dashboard to display the metrics of the different data releases in Open Targets.")
-st.sidebar.title("What do you want to check?")
+
+st.sidebar.header("What do you want to do?")
+page = st.sidebar.radio(
+    "Choose an option",
+    ("Explore metrics", "Compare metrics", "Statistics"),
+    index=1,
+    help="Explore metrics allows you to visualise and filter the metrics of the selected datasets. Compare metrics allows you to compare the main metrics between two releases."
```

Comment:
Wrapping line length to 120 characters:

```suggestion
    help=("Explore metrics allows you to visualise and filter the metrics of the selected datasets. Compare metrics "
          "allows you to compare the main metrics between two releases.")
```

Reply:
Fixed in `025ba22`